core/controllers/contributor_dashboard.py

```diff
@@ -524,3 +524,46 @@ def get(self):
             'topic_names': topic_names
         }
         self.render_json(self.values)
+
+
+class TranslationPreferenceHandler(base.BaseHandler):
+    """Provides the preferred translation language in the
+    contributor dashboard page.
+    """
+
+    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+    URL_PATH_ARGS_SCHEMAS = {}
+    HANDLER_ARGS_SCHEMAS = {
+        'GET': {},
+        'POST': {
+            'language_code': {
+                'schema': {
+                    'type': 'basestring',
+                    'validators': [{
+                        'id': 'is_supported_audio_language_code'
```

Comment:
This is a bit confusing, should we rename the function to smth like `is_supported_language_code`?

Reply:
As of now, we are using `SUPPORTED_AUDIO_LANGUAGES` for both written translations and voice-over translations and also for setting the audio language for the exploration. `is_supported_language_code` uses `SUPPORTED_AUDIO_LANGUAGES` to validate. If we rename this validator then I believe we need to make changes to a lot of places. I think this will be out of scope for this PR, we can create a new issue for the renaming of all these validators and make sure that the name clearly defines the purpose.

Reply:
I have created a new issue.
Issue Link: https://github.com/oppia/oppia/issues/15559

Reply:
I have added a TODO comment just above the "HANDLER_ARGS_SCHEMAS".